libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -100,6 +100,10 @@ void AP_Logger_File::Init()
 
     hal.console->printf("AP_Logger_File: buffer size=%u\n", (unsigned)bufsize);
 
+    // We got this far with logger, this is perfect spot
```

Comment:
This should probably go into `AP_Vehicle::setup()`.  AFAICS it doesn't have anything to do with dataflash logging.

Reply:
ok, I will do that, this location just looked welcoming...